bin/proxy.js

```diff
@@ -52,31 +55,46 @@ function getHost(req) {
   return frontendHost;
 }
 
-const server = http.createServer((req, res) => {
+// Skip SSL check to avoid the 'self signed certificate in certificate chain' error.
+const proxy = httpProxy.createProxyServer({ secure: false });
+
+const handler = (req, res) => {
   const host = getHost(req);
   return proxy.web(req, res, {
     target: host,
     changeOrigin: true,
     autoRewrite: true,
-    protocolRewrite: 'http',
+    protocolRewrite: protocol,
```

Comment:
Hmm, does that work when proxying non-https URLs? I guess it would only come into play if we tried to proxy a local Docker API via `olympia.test`. No one really uses that so it's not a big deal.

Reply:
I don't know, before it was `http` only so I don't know.. Maybe we could remove this option if we get an issue?

And if no one is using the `olympia.test` env (docker env, no?), then we could maybe remove it entirely?

Reply:
Yes, `olympia.test` is for Docker and we want to support it even though no one uses it. It's the only way you can develop without Internet access. However, I'm not worried about your change on this line. Let's see what happens.

Reply:
I tried to develop without Internet access last time... Ended up writing a Go proxy to cache everything before going offline -_-